backend/service/auditsink/slack/slack_helper.go

```diff
@@ -0,0 +1,46 @@
+package slack
+
+import (
+	"fmt"
+
+	configv1 "github.com/lyft/clutch/backend/api/config/service/auditsink/slack/v1"
+)
+
+// NewOverrideLookup creates a map from the overrides. The generated OverrideLookup is
+// used to easily retrieve an override by key (/service/method).
+func NewOverrideLookup(overrides []*configv1.CustomMessage) *OverrideLookup {
+	if len(overrides) == 0 {
+		// no overrides
+		return nil
+	}
```

Comment:
i would return an empty struct instead of nil. if OverrideLookup gets more complicated it is bug prone to have to remember to nil check in every func.

alternatively you could deal in the value, e.g. return OverrideLookup instead of pointer. i know in Clutch we normally default to using pointers for all structs, but in this case it would be fine (and probably more efficient) to use the value, especially where the members of the struct are reference types as the map is here.

Reply:
thanks, yeah agree. in latest commit went with the second approach and also updated `GetOverrideMessage` to have a value receiver since the func does not modify the OverrideLookup map